_sass/_base.scss

```diff
@@ -14,77 +24,141 @@ header img {
 
 img.logo {
   display: block;
-  margin: 0 auto;
+  margin: 0.25em 0;
   padding: 1rem;
   max-width: 6rem;
 }
 
-h1, h2, h3, h4, h5 {
-  font-family: 'Lato', sans-serif;
-  margin-left: 1rem;
+h2, h3, h4, h5 {
+  font-size: 4.8rem;
   margin-right: 1rem;
+  @media(max-width: 670px){
+    font-size: 3rem;
+  }
+}
+h1 {
+  margin-top: 0; margin-bottom: 1em;
+}
+h2 {
+  margin-top: 2em; margin-bottom: 0.25rem;
+}
+h2, strong {
+  font-size: 4.5rem;
+  @media(max-width: 670px){
+    font-size: 3rem;
+  }
+}
+h4 {
+  margin-top: 0; margin-bottom: 0;
 }
-h1 { margin-top: 0 }
-h2 { margin-bottom: 0.25rem; }
-h4 { margin-top: 0; margin-bottom: 0; }
 p, li {
-  font-size: 1.1rem;
+  font-size: 4.8rem;
+  line-height: 0.95;
+  @media(max-width: 670px){
+    font-size: 3rem;
+  }
+  // font-weight: bold;
 }
 p {
-  margin-left: 1rem;
   margin-right: 1rem;
+  margin-bottom: 0;
+  font-family: 'Times', serif;
+  &:first-of-type {
+    margin-top: 0;
+  }
+  & + p {
+    text-indent: 3em;
+    margin-top: 0;
+    @media(max-width: 670px) {
+      text-indent: 2em;
+    }
+  }
 }
-li {
-  margin-bottom: 1rem;
+
+br + p {
+  margin-top: 0;
+}
+
+ol {
+  margin-top: 0;
+  margin-left: 4.5em;
+  max-width: 80%;
+
+  @media(max-width: 670px) {
+    margin-left: 2em;
+  }
+
+  li {
+    font-family: 'Times';
+  }
 }
 ul {
   margin-top: 2rem;
+  list-style: none;
+  li {
+    padding: 0;
+    position: relative;
+    &:before {
+      content: '\2B24';
+      font-size: 3.1rem;
+      position: absolute;
+      left: -1.5em; top: 0.15em;
+
+      @media(max-width: 670px) {
```

Comment:
we should probably make the media widths a constant variable rather than repeating, that said this is such a small site it probably does not matter too much.